modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -4,26 +4,23 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set-aws_{context}"]
+= Adding a multi-architecture compute machine set to your AWS cluster
 
-= Adding an ARM64 compute machine set to your cluster
-
-To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster so that your cluster has multi-architecture compute machines.
+To configure an AWS cluster with multi-architecture compute machines, you must create an AWS ARM64 or AMD64 compute machine set. This adds ARM64 or AMD64 compute nodes to your cluster.
```

Comment:
Instead of **AMD64** and **ARM64**, we prefer to use **64-Bit ARM** and **64-bit x86**

Reply:
also, you can remove: `This adds ARM64 or AMD64 compute nodes to your cluster`

Reply:
Updated as per the suggestion.